### PR TITLE
enroll class

### PR DESCRIPTION
**Pull request by widyaap533**

### CODE REVIEW

**Comment by widyaap533** [comment]
...

**Comment by Ohwdyaa** [comment]
p

File: src/main/java/org/binar/pragosacademyapi/service/CourseService.java
Comment (Gunawann20) [inline src/main/java/org/binar/pragosacademyapi/service/CourseService.java:1]: Ganti jadi Response<String> enrollCourse(String courseCode, PaymentRequest request)

Buat kelas PaymentRequest di package request atributnya 
    private String cardNumber;
    private String cardHolderName;
    private String cvv;
    private String expiryDate;

File: src/main/java/org/binar/pragosacademyapi/service/impl/CourseServiceImpl.java
Comment (Gunawann20) [inline src/main/java/org/binar/pragosacademyapi/service/impl/CourseServiceImpl.java:1]: - Cek entity payment ada update atribut
- Set statusnya langsung true aja

**Review by Gunawann20 — COMMENTED** [review]
Revisi sesuai komen

File: src/main/java/org/binar/pragosacademyapi/service/CourseService.java
Comment (Gunawann20) [reply src/main/java/org/binar/pragosacademyapi/service/CourseService.java:1]: Response<String> enrollCourse(String courseCode, PaymentRequest request)

Comment (Gunawann20) [reply src/main/java/org/binar/pragosacademyapi/service/CourseService.java:1]: `Response<String> enrollCourse(String courseCode, PaymentRequest request);`

**Pull request merged**